sdk/include/opentelemetry/sdk/logs/recordable.h

```diff
@@ -92,6 +92,14 @@ class Recordable
    * @param trace_flags the trace flags to set
    */
   virtual void SetTraceFlags(opentelemetry::trace::TraceFlags trace_flags) noexcept = 0;
+
+  /**
+   * Set instrumentation_library for this log.
+   * @param instrumentation_library the instrumentation library to set
+   */
+  virtual void SetInstrumentationLibrary(
+      const opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary
+          &instrumentation_library) noexcept = 0;
```

Comment:
We need to call this method from within Logger::Log() for exporters to be able to read this. Do we plan to have it as separate PR?